backend/dataall/modules/mlstudio/api/queries.py

```diff
@@ -0,0 +1,49 @@
+"""The module defines GraphQL queries for the SageMaker ML Studio"""
+from dataall.api import gql
+from dataall.modules.mlstudio.api.resolvers import (
+    get_sagemaker_studio_user,
+    # get_user_applications,
+    list_sagemaker_studio_users,
+    get_sagemaker_studio_user_presigned_url,
+)
+
+getSagemakerStudioUser = gql.QueryField(
+    name='getSagemakerStudioUser',
+    args=[
+        gql.Argument(
+            name='sagemakerStudioUserUri', type=gql.NonNullableType(gql.String)
+        )
+    ],
+    type=gql.Ref('SagemakerStudioUser'),
+    resolver=get_sagemaker_studio_user,
+)
+
+# TODO: is this used????
+# getSagemakerStudioUserApps = gql.QueryField(
+#     name='getSagemakerStudioUserApps',
```

Comment:
In the repository - it's not. I believe it can be safely deleted

Reply:
Done